### PR TITLE
Refactored the scss for the alerts.

### PR DESCRIPTION
**Pull request by klje**

### CODE REVIEW

**Comment by klje** [comment]
@jenkins-plone-org please run jobs

File: scss/alerts.scss [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  border-radius: 0.25rem;
+  animation-fill-mode: forwards !important;
+  animation-delay: 2s !important;
+  transition: border-width 0.6s linear;
+  a {
+      font-weight: 700;
+      position: relative;
+      text-decoration: none;
+    }
+}
```
Comment (MrTango) [inline scss/alerts.scss:14]: this might work, but i still think it would be better to wrap the whole definitions in one .alert{} block ;)

**Review by MrTango — APPROVED** [review]
LGTM, please do a rebase from master to fix the conflicts with the compiled css files and merge the PR.

**Comment by jensens** [comment]
> LGTM, please do a rebase from master to fix the conflicts with the compiled css files and merge the PR.

@klje with a PR in this package, it is better to no include the compiled changes, but only the SCSS - otherwise its gets annoying for you.

**Review by agitator — CHANGES_REQUESTED** [review]
Please read the comments in https://github.com/plone/plonetheme.barceloneta/pull/245#issuecomment-899677498

**Pull request merged**